<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lendDialog</class>
 <widget class="QDialog" name="lendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lending</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtRc.qrc">
    <normaloff>:/image/images/lend.png</normaloff>:/image/images/lend.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="idLine">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="lendButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="text">
      <string>Lend</string>
     </property>
     <property name="icon">
      <iconset resource="QtRc.qrc">
       <normaloff>:/image/images/lend.png</normaloff>:/image/images/lend.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="lendToReserveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="text">
      <string>Lend to reserved user</string>
     </property>
     <property name="icon">
      <iconset resource="QtRc.qrc">
       <normaloff>:/image/images/lend_to_reserved.png</normaloff>:/image/images/lend_to_reserved.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="reserveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="text">
      <string>Reserve</string>
     </property>
     <property name="icon">
      <iconset resource="QtRc.qrc">
       <normaloff>:/image/images/reserve.png</normaloff>:/image/images/reserve.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="takeBackButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="text">
      <string>Take Back</string>
     </property>
     <property name="icon">
      <iconset resource="QtRc.qrc">
       <normaloff>:/image/images/takeback.png</normaloff>:/image/images/takeback.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtRc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>idLine</sender>
   <signal>returnPressed()</signal>
   <receiver>lendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
